templates/snippets/carousel.html

```diff
@@ -0,0 +1,41 @@
+{% if updates %}
+<div id="carousel">
+    {% for post in updates %}
+    {% if forloop.first %}
+    <div class="post-update active">
```

Comment:
Why not wrap just the `active` attribute in the conditional instead of repeating the whole line?